trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -39,286 +40,582 @@ def plot_vehicle(vehicle, save_figure = False, plot_control_points = True, save_
     N/A	
     """	
     # unpack vortex distribution 
-    VD = vehicle.vortex_distribution
-    
-    face_color = 'grey'        
-    edge_color = 'grey'
-    alpha_val  = 0.5  
-    
+    try:
+        VD = vehicle.vortex_distribution 
+    except:
+        settings = Data()
+        settings.number_spanwise_vortices  = 25
+        settings.number_chordwise_vortices = 5
+        VD = generate_wing_vortex_distribution(vehicle,settings)  
+        
     # initalize figure 
     fig = plt.figure(save_filename) 
-    fig.set_size_inches(8, 8) 
+    fig.set_size_inches(12, 12) 
     axes = Axes3D(fig)    
-    axes.view_init(elev= 30, azim= 210)   
+    axes.view_init(elev= 30, azim= 210)     
+    
+    # -------------------------------------------------------------------------
+    # PLOT WING
+    # -------------------------------------------------------------------------
+    wing_face_color = 'grey'        
+    wing_edge_color = 'lightgrey'
+    wing_alpha_val  = 1    
+    plot_wing(axes,VD,wing_face_color,wing_edge_color,wing_alpha_val)  
+    if  plot_control_points:
+        axes.scatter(VD.XC,VD.YC,VD.ZC, c='r', marker = 'o' )
+    
+    # -------------------------------------------------------------------------
+    # PLOT WAKE
+    # -------------------------------------------------------------------------    
+    wake_face_color = 'white'                
+    wake_edge_color = 'blue' 
+    wake_alpha      = 0.5  
+    if'Wake' in VD:
+        plot_propeller_wake(axes, VD,wake_face_color,wake_edge_color,wake_alpha)
+        
+    # -------------------------------------------------------------------------
+    # PLOT FUSELAGE
+    # -------------------------------------------------------------------------        
+    fuselage_face_color = 'grey'                
+    fuselage_edge_color = 'black' 
+    fuselage_alpha      = 1      
+    for fus in vehicle.fuselages:     
+        plot_fuselage(axes, fus ,fuselage_face_color,fuselage_edge_color,fuselage_alpha)     
+  
+    # -------------------------------------------------------------------------
+    # PLOT ENGINE
+    # -------------------------------------------------------------------------        
+    propulsor_face_color = 'darkred'                
+    propulsor_edge_color = 'black' 
+    propulsor_alpha      = 1    
+    for propulsor in vehicle.propulsors:    
+        plot_propulsor(axes,VD,propulsor,propulsor_face_color,propulsor_edge_color,propulsor_alpha)    
+      
+    # Plot Vehicle
+    plt.axis('off') 
+    plt.grid(None)      
+    return 
+
+def plot_wing(axes,VD,face_color,edge_color,alpha_val): 
+    """ This plots the wings of a vehicle 
+
+    Assumptions: 
+    None
+
+    Source:   
+    None
+    
+    Inputs:   
+    VD                   - vortex distribution    
```

Comment:
Fill these out, otherwise I'd assume alpha is AoA

Reply:
It is actually the degree of transparency. Updated headers.